test/snippet/io/alignment_file/alignment_file_output_filename_construction_with_ref_info.cpp

```diff
@@ -0,0 +1,15 @@
+#include <string>
+#include <vector>
+
+#include <seqan3/io/alignment_file/output.hpp>
+#include <seqan3/std/filesystem>
+
+int main()
+{
+    auto tmp_dir = std::filesystem::temp_directory_path();
+
+    std::vector<std::string> ref_ids{"ref1", "ref2"};
+    std::vector<size_t>      ref_lengths{1234, 5678};
+
+    seqan3::alignment_file_output fout{tmp_dir/"my.sam", ref_ids, ref_lengths};
```

Comment:
please write into stringstream

Reply:
Should they still use stringstream if they are snippets for "Construct from filename"?

Reply:
Also, this snippet is referenced as such:
"     * In most cases the template parameters are deduced completely automatically:
     *
     * \include test/snippet/io/alignment_file/alignment_file_output_filename_construction_with_ref_info.cpp"

But if I construct from stream I don't think the template parameter deduction works...